README.md

```diff
@@ -801,6 +805,61 @@ For the vast majority of cases this will get you most of the migration:
 - add `import PropTypes from 'prop-types'` (Webpacker only)
 - re-run `bundle exec rails webpacker:install:react` to update npm packages (Webpacker only)
 
+## Migrating from `react-rails` to `react_on_rails`
+
+### Why to migrate?
+
+[`react_on_rails`](https://github.com/shakacode/react_on_rails/) offers several additional features for a Rails + React application. The following is a table of features comparison.
+
+| **Feature**             | **react-rails** | **react-on-rails** |
+| ----------------------- |:---------------:|:------------------:|
+| Sprockets               | ✅               | ❌                  |
+| Shakapacker             | ✅               | ✅                  |
+| SSR                     | ✅               | ✅                  |
+| SSR with HMR            | ✅               | ✅                  |
+| SSR with React-Router   | ❌               | ✅                  |
+| SSR with Code Splitting | ❌               | ✅                  |
+| Node SSR                | ❌               | ✅                  |
```

Comment:
Sorry to say that now it's been merged, but it doesn't bother you to not mention that SSR with react-on-rails is not free?